.prettierrc

```diff
@@ -1,5 +1,13 @@
 {
   "singleQuote": true,
   "tabWidth": 2,
-  "trailingComma": "all"
-}
\ No newline at end of file
+  "trailingComma": "all",
+  "overrides": [
+    {
+      "files": "**/scripts/*",
```

Comment:
Make the list explicit (`packages/braid-design-system/scripts/generateIcons.cts`)